### DISCUSSION TITLE
A question about reading ABAQUS INP

### DISCUSSION THREAD

**Discussion by DanielChou0916** [idaholab/moose#26089] [Q&A Meshing] [answered]
I'm a beginner of  MOOSE, and I'm trying to practice read the mesh from ABAQUS inp in MOOSE.
In the directory "\projects\moose\test\tests\mesh\abaqus_input"
I saw the files

> testcube_elem_id_in.inp
> testcube_elem_id.i
> testcube_elem_id_in.e

I was wondering does this mean we can use ''testcube_elem_id.i'' to generate a mesh file ''testcube_elem_id_in.e'' from "testcube_elem_id_in.inp"?
If yes, how do we do that?
The testcube_elem_id.i only has codes as follows:
```
[Mesh]
  type = FileMesh
  file = testcube_elem_id_in.inp
[]
```
When I create a new application named 'abaqus_mesh_conversion', and copy "testcube_elem_id.i" and "testcube_elem_id_in.inp" into a problems folder, then execute:
`~/projects/abaqus_mesh_conversion/abaqus_mesh_conversion-opt -i testcube_elem_id.i`
It leads to the error message:

> *** ERROR ***
"Executioner" does not exist, make sure your input file contains an [Executioner] block or your simulation adds an Executioner through an Action.

> Stack frames: 13
> 0: libMesh::print_trace(std::ostream&)
> 1: moose::internal::mooseErrorRaw(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)
> 2: void mooseError<char const (&) [146]>(char const (&) [146])
> 3: CheckIntegrityAction::act()
> 4: Action::timedAct()
> 5: ActionWarehouse::executeActionsWithAction(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
> 6: ActionWarehouse::executeAllActions()
> 7: MooseApp::runInputFile()
> 8: MooseApp::run()
> 9: main
> 10: /lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7f26b0548d90]
> 11: __libc_start_main
> 12: /home/fem/projects/abaqus_mesh_conversion/abaqus_mesh_conversion-opt(+0x33ef) [0x556bbcaeb3ef]
> application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
> [unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
> :
> system msg for write_line failure : Bad file descriptor

I believe that they might not work like this, but I have no idea about how they work correctly.
Could anyone give me some hints?
Appreciate for patience :))

**Comment by GiudGiud** [comment]
Hello

You ll have to add `--mesh-only` (on the command line) to run a meshing-only simulation

Guillaume

**Reply by GiudGiud** [reply, marked as answer]
so
```
~/projects/abaqus_mesh_conversion/abaqus_mesh_conversion-opt -i testcube_elem_id.i --mesh-only
```

**Reply by DanielChou0916** [reply]
Sincerely appreciate for your indication :)))))
It works well now
